examples/compose/column_transformer_mixed_types.py

```diff
@@ -0,0 +1,104 @@
+"""
+===================================
+Column Transformer with Mixed Types
+===================================
+
+This example illustrates how to apply different preprocessing and
+feature extraction pipelines to different subsets of features,
+using :class:`sklearn.compose.ColumnTransformer`.
+This is particularly handy for the case of datasets that contain
+heterogeneous data types, since we may want to scale the
+numeric features and one-hot encode the categorical ones.
+
+In this example, the numeric data is standard-scaled after
+mean-imputation, while the categorical data is one-hot
+endcoded after imputing missing values with a new category
+(:code:`'missing'`).
```

Comment:
What is the `:code:` doing? (I haven't seen this sphinx role before) 
I think it can simply be double backticks

Reply:
Same as double backticks, yep. Will use the latter.